charts/nginx-gateway-fabric/templates/nginxproxy.yaml

```diff
@@ -19,7 +19,9 @@ spec:
       {{- end }}
       container:
         {{- if .Values.nginx.container }}
```

Comment:
I don't think you should need this `if` statement (or below) if you're using `with`.